src/services/env.js

```diff
@@ -4,8 +4,12 @@ const path = require('path');
 const { CLI_NAME } = require('./config');
 
 const configureEnv = () => {
-  const home = process.env.HOME || process.env.USERPROFILE || os.homedir();
-  const twilioDir = path.join(home, `.${CLI_NAME}`);
+  const twilioDir = (() => {
+    if (process.platform === 'win32' && process.env.APPDATA) {
+      return path.join(process.env.APPDATA, CLI_NAME);
+    }
+    return path.join(process.env.HOME || os.homedir(), `.${CLI_NAME}`);
```

Comment:
Shouldn't we still have `process.env.USERPROFILE` here?

Reply:
Yes, you are right, I'll add it back

Reply:
Thank you!